xmtp_mls/src/groups/intents.rs

```diff
@@ -501,7 +501,7 @@ mod tests {
         let intent =
             UpdateGroupMembershipIntentData::new(membership_updates, vec!["bar".to_string()]);
 
-        let as_bytes: Vec<u8> = intent.clone().try_into().unwrap();
+        let as_bytes: Vec<u8> = intent.clone().into();
```

Comment:
Will this end up being the same code?

Reply:
Yeah, `TryFrom` is auto-impl for anything that impls `From`